app/assets/stylesheets/application.scss

```diff
@@ -23,9 +23,9 @@
 
 @charset "UTF-8";
 
-.full-height {
-  height: calc(100vh - 120px);
-}
+// .full-height {
+//   height: calc(100vh - 120px);
```

Comment:
I have changed in this in my branch which I will add soon.

We need to have the min-height. It will work for both our screens.